src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -123,6 +125,18 @@ public void Load(string settingsXml)
 #endif
         }
 
+        public void SaveRandomSeed(string dirname)
+        {
+            using (var writer = new StreamWriter(Path.Combine(dirname, "$RANDOM_SEED$")))
```

Comment:
Good catch. I intended to take another look for error handling after coding the happy pathg but didn't... working too fast!